doc/articles/features/ElevatedView.md

```diff
@@ -0,0 +1,47 @@
+# ElevatedView
+On many design language like _Material Design_, there's a notion of elevation where a portion of the UI should be presented as been _elevated_ over the rest of the content.
+
+In this case, the UWP proposed solution can't work on all platforms because of technical limitations. To address this problem, Uno provides a control called `ElevatedView`, able to produce a similar result on all platforms, providing an elevated result.
+
+This control is very useful to create cards with both rounded corners and elevated effect - which could be challenging to produce on some platforms.
+
+## How to use the `ElevatedView`
+
```

Comment:
First step is to have the nuget reference to Uno or Uno.UI.Toolkit, it's often forgotten on UWP.